src/datasets/utils/metadata.py

```diff
@@ -218,8 +218,10 @@ class _nothing:
 class DatasetMetadata:
     annotations_creators: List[str]
     language_creators: Union[EmptyList, List[str]]
-    languages: Union[EmptyList, List[str]]
-    licenses: List[str]
+    language: Union[EmptyList, List[str]]
+    languages: Union[EmptyList, List[str]]  # deprecated
+    license: List[str]
+    licenses: List[str]  # deprecated
```

Comment:
I think you can simply give default values (an empty list should do the job) to `languages` and `licenses` to fix the CI